code/modules/mob/living/simple_animal/abnormality/aleph/blue_star.dm

```diff
@@ -108,3 +110,9 @@
 	forceMove(T)
 	BluePulse()
 	return
+
+/ego_gifts/star
```

Comment:
I'd say all ego gifts should be in a separate folder.

Reply:
Done~